front/app/components/SignUpIn/SignUpInModal.tsx

```diff
@@ -60,7 +60,9 @@ const SignUpInModal = memo<Props>(({ className, onMounted }) => {
     : 580;
 
   const modalNoClose = !!(
-    metaData?.error !== true && signUpActiveStep === 'verification'
+    (metaData?.error !== true && signUpActiveStep === 'verification') ||
+    (signUpActiveStep &&
+      metaData?.modalNoCloseSteps?.includes(signUpActiveStep))
```

Comment:
Looks good to me in case we would need this in more places, but right now I'm wondering if just replacing this whole condition with `signUpActiveStep === 'verification'` wouldn't be easier.

`const modalNoClose = signUpActiveStep === 'verification';`

Reply:
You mean `confirmation` right? It would be easier indeed but it would be a leakage from the confirmation module into the core.

In this case, the impact of going with your suggestion is minimal since the module is free, but for other cases, a solution like this will have to be used.

Reply:
Oh, I think I was indeed confusing 'verification' and 'confirmation'. Never mind my comment :-)